File: LUYENCODE/CONTETST/CƠ BẢN 2/DK09 - Kiểm tra năm nhuận/DK09.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    cin >> n;
    if(n > 0 && n < 100000)
    {
        if((n % 4 == 0 || n % 400 == 0) && n % 100 != 0)
            cout << "YES";
        else
            cout << "NO";
    }
    else
        cout << "INVALID";
    return 0;
}

File: LUYENCODE/CONTETST/CƠ BẢN 2/DK09 - Kiểm tra năm nhuận/DK09.py
n = int(input())
if 0 < n < 10000:
    if (n % 4 == 0 or n % 400 == 0) and n % 100 != 0:
        print("YES", end="")
    else:
        print("NO", end="")
print("INVALID", end="")